testCommon/src/test/resources/setup-for-diverged-and-older-than.sh

```diff
@@ -60,6 +60,8 @@ create_repo machete-sandbox
   sed 's/^  //' <<< "$machete_file" > .git/machete
 )
 
+sleep 1
```

Comment:
Hmm how about using `set_fake_git_date` instead to achieve the same effect in this script?